chart/epinio/templates/ingress.yaml

```diff
@@ -16,13 +16,44 @@ spec:
   - host: "epinio.{{ .Values.global.domain }}"
     http:
       paths:
+  {{- if ".Values.epinio-ui.enabled" }}
+      - backend:
+          service:
+            name: epinio-ui
+            port:
+              number: 80
+        path: /
+        pathType: ImplementationSpecific
+      - backend:
+          service:
+            name: epinio-server
+            port:
+              number: 80
+        path: /api
+        pathType: ImplementationSpecific
+      - backend:
+          service:
+            name: epinio-server
+            port:
+              number: 80
+        path: /wapi
+        pathType: ImplementationSpecific
+      - backend:
+          service:
+            name: epinio-server
+            port:
+              number: 80
+        path: /ready
+        pathType: ImplementationSpecific
+  {{- else }}
```

Comment:
Any reason why we can't limit Epinio endpoints to `/api` and `/wapi` always? In that case, we would simply add the `/` endpoint when `epinio-ui` is enabled.